src/components/ReplSection.js

```diff
@@ -0,0 +1,105 @@
+import React from "react"
+import {
+  AspectRatio,
+  Box,
+  Container,
+  Heading,
+  Text,
+  SimpleGrid,
+  Stack,
+} from "@chakra-ui/react"
+
+import { Prism as SyntaxHighlighter } from "react-syntax-highlighter"
+import { a11yDark } from "react-syntax-highlighter/dist/cjs/styles/prism"
+
+//TODO: We could remove xarrray installation step by requesting xarray to be added to the Pyodide standard lib
```

Comment:
This seems like a reasonable thing to ask? What's the process of adding a package to the standard lib?

Reply:
we need to open an issue/pr in https://github.com/pyodide/pyodide